test/utils/base_page/base_page.js

```diff
@@ -0,0 +1,23 @@
+const logger = require('../../../test/config/logger.config.js');
+const Header = require('./header');
+
+class BasePage {
+    constructor() {
+        this.Header = new Header();
+    };
+    wait(seconds) {
```

Comment:
Please don't break the Single responsibility principle here: move the `wait()` method to a separate function helper.

Reply:
done